docs/source/usage/parameters.rst

```diff
@@ -72,6 +72,31 @@ Domain Boundary Conditions
 
    TODO :-)
 
+Lattice elements
+----------------
```

Comment:
@cemitch99 @qianglbl Feel free to suggest a better phrasing for the documentation below.
In particular, for the quad, maybe we should mention whether a positive `k` is focusing along `x` or along `y`.
For the bend, should we also specify along which direction it is bending the beam?

Note that, once the PR is merged, the text below will automatic appear on this page:
https://impactx.readthedocs.io/en/latest/usage/parameters.html

Reply:
It would probably be good to specify it in the document. (that should probably follow the same notation as other codes like MAD.